components/kyma-metrics-collector/pkg/process/process.go

```diff
@@ -284,21 +299,26 @@ func (p Process) getRecordWithOldOrNewMetric(identifier int, subAccountID string
 		}
 		return oldRecord, true, nil
 	}
+	gardenerErrorCount.Reset()
+	skrErrorCount.Reset()
 	return &record, false, nil
 }
 
 func (p Process) sendEventStreamToEDP(tenant string, payload []byte) error {
 	edpRequest, err := p.EDPClient.NewRequest(tenant)
 	if err != nil {
+		edpErrorCount.WithLabelValues("failed_request", tenant).Inc()
```

Comment:
The name of this metric could be improved. 'failed_request' could mean, that the request fails, but in this case it means that creating the request object failed.

Reply:
changed label to "failed_createRequest"